Core/Transaction.cs

```diff
@@ -1,47 +1,49 @@
-using Newtonsoft.Json;
 using System;
+using Newtonsoft.Json;
 
 namespace Core
 {
-    public class Transaction : IComparable<Transaction>
-    {
-        public string CardNumber { get; }
-        public string ApplicationCode { get; }
-        public DateTimeOffset TimeStamp { get; }
-        public Money Amount { get; }
-        public bool IsIncome { get; }
-        public bool IsExpence { get; }
-        public Money Rest { get; }
-        public string Terminal { get; }
-        public string Description { get; }
+	public class Transaction : IComparable<Transaction>
+	{
+		public Transaction(string cardNumber, string appCode, DateTimeOffset timeStamp, Money amount, Money rest,
+		                   string terminal, string description)
+		{
+			CardNumber = cardNumber;
+			ApplicationCode = appCode;
+			TimeStamp = timeStamp;
+			Amount = amount.Amount >= 0 ? amount : new Money(-amount.Amount, amount.Currency);
+			IsIncome = amount.Amount > 0;
+			IsExpence = amount.Amount < 0;
+			Rest = rest;
+			Terminal = terminal;
+			Description = description;
+		}
 
-        public Transaction(string cardNumber, string appCode, DateTimeOffset timeStamp, Money amount, Money rest, string terminal, string description)
-        {
-            CardNumber = cardNumber;
-            ApplicationCode = appCode;
-            TimeStamp = timeStamp;
-            Amount = amount.Amount >= 0 ? amount : new Money(-amount.Amount, amount.Currency);
-            IsIncome = amount.Amount > 0;
-            IsExpence = amount.Amount < 0;
-            Rest = rest;
-            Terminal = terminal;
-            Description = description;
-        }
+		[JsonConstructor]
+		public Transaction(string cardNumber, string applicationCode, DateTimeOffset timeStamp, Money amount,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constructor has 9 parameters, which is greater than the 7 authorized.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)